src/test/java/mocacong/server/domain/BaseTimeTest.java

```diff
@@ -0,0 +1,66 @@
+package mocacong.server.domain;
+
+import mocacong.server.domain.cafedetail.*;
+import mocacong.server.exception.notfound.NotFoundCafeException;
+import mocacong.server.repository.CafeRepository;
+import mocacong.server.repository.MemberRepository;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+import javax.persistence.EntityManager;
+import java.time.LocalDateTime;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+@DataJpaTest
```

Comment:
DataJpaTest는 SpringBootTest와 다르게 특정 빈들만 가져옵니다.
```suggestion
@DataJpaTest
@Import(JpaAuditingConfiguration.class)
```